examples/nidaqmx/analog-input-every-n-samples.py

```diff
@@ -169,7 +169,7 @@ async def wait_for_done():
             check_for_warning(stop_task_response)
 
         except grpc.RpcError as rpc_error:
-            error_message = rpc_error.details()
+            error_message = str(rpc_error.details() or "")
```

Comment:
Same change in every file. This makes sure error_message is a string because before when it tried to do `error_message += f"\nError status: {details}"` mypy v1.2.0 was complaining that `error: Unsupported left operand type for + ("None")  [operator]`